libraries/aws_config_recorder.rb

```diff
@@ -54,12 +58,14 @@ def status
     catch_aws_errors do
       @status_response = @aws.config_client.describe_configuration_recorder_status(configuration_recorder_names: [@recorder_name])
       @status = @status_response.configuration_recorders_status.first.to_h
+      @last_status = @status[:last_status]
     end
   end
 
   def recording?
     return false unless exists?
-    status[:recording]
+    @recording = @status[:recording]
+    @recording
```

Comment:
```suggestion
    @status[:recording]
```

Reply:
1. Done
2. Done
3. Following is the explanation for ONLY SINGULAR for this resource. I have already got it verified from SAP and they are fine with it. _"While working on its plural versions, I was checking AWS documentation and found that there can be only one configuration-recorder allowed per region. This may be the reason plural version of it was skipped in the initial implementation. So having plural version may not be applicable for this resource."_